firefox-ios/Client/Frontend/Autofill/Address/Edit/EditAddressViewController.swift

```diff
@@ -9,27 +9,13 @@ import Common
 import struct MozillaAppServices.UpdatableAddressFields
 
 class EditAddressViewController: UIViewController, WKNavigationDelegate, WKScriptMessageHandler, Themeable {
-    private lazy var webView: WKWebView = {
-        let webConfiguration = WKWebViewConfiguration()
-        let webView = WKWebView(frame: .zero, configuration: webConfiguration)
-        webView.translatesAutoresizingMaskIntoConstraints = false
-        webView.navigationDelegate = self
-        return webView
-    }()
-
-    private lazy var activityIndicator: UIActivityIndicatorView = {
-        let activityIndicator = UIActivityIndicatorView(style: .large)
-        activityIndicator.hidesWhenStopped = true
-        activityIndicator.translatesAutoresizingMaskIntoConstraints = false
-        return activityIndicator
-    }()
-
     var model: AddressListViewModel
     private let logger: Logger
     var themeManager: ThemeManager
     var themeObserver: NSObjectProtocol?
     var notificationCenter: NotificationProtocol = NotificationCenter.default
     var currentWindowUUID: WindowUUID? { model.windowUUID }
+    lazy var editAddressWebViewManager = model.editAddressWebViewManager
```

Comment:
I think we can do something like var webView: WKWebView? { model.editAddressWebViewManager.webView }, and everywhere else to check only guard let webView else { return }

Reply:
Updated in https://github.com/mozilla-mobile/firefox-ios/pull/20738/commits/9e98f326ab818d5529c19f91c0a4e2464b67bac0